### PR TITLE
Adding Sejda Desktop

### PR DESCRIPTION
**Pull request by ediweissmann**
Electron was immensely helpful for us building a modern, cross platform desktop application that reused a lot of the web technology we already developed for our cloud services. Thank you!

[Sejda Desktop Homepage](https://www.sejda.com/desktop)

> If you're submitting a closed source app, include evidence of it being built with Electron.

screen shot 2017-03-09 at 10 31 13

### CODE REVIEW

File: apps/sejda-pdf-desktop/sejda-pdf-desktop.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: 'Sejda PDF Desktop'
+description: 'Pleasant and productive PDF software that you''ll love to use'
```
Comment (zeke) [inline apps/sejda-pdf-desktop/sejda-pdf-desktop.yml]: you''ll → you'll

Comment (ediweissmann) [inline apps/sejda-pdf-desktop/sejda-pdf-desktop.yml]: I think that's the yaml escape for the `'` char.

Comment (zeke) [inline apps/sejda-pdf-desktop/sejda-pdf-desktop.yml]: Ah I see.

Would you mind using double quotes anyway? "Pleasant and productive PDF software that you'll love to use" just to be sure this doesn't cause any problems down the line.

Comment (ediweissmann) [inline apps/sejda-pdf-desktop/sejda-pdf-desktop.yml]: Sure, no worries.

**Comment by zeke** [comment]
Thanks!

**Pull request merged**